### ISSUE TITLE
Checkboxes in variable box: Missing semantics and accessibility in list

### ISSUE THREAD

**Issue by Maren-SSB** [PxTools/pxtools-project#59]
https://statistics-norway.atlassian.net/browse/PXWEB2-1091?atlOrigin=eyJpIjoiZDRlNjc2ZWIxZjY3NDI1MGFhYTg4Y2RjYTQ2MjllOWUiLCJwIjoiaiJ9
